spacy/cli/package.py

```diff
@@ -112,7 +112,9 @@ def package(
         msg.fail("Invalid pipeline meta.json")
         print("\n".join(errors))
         sys.exit(1)
-    model_name = meta["lang"] + "_" + meta["name"]
+    model_name = meta["name"]
+    if not model_name.startswith(meta['lang']):
```

Comment:
Maybe `model_name.startswith(meta['lang'] + "_")` instead?

Overall I'm a little worried this change is going to cause too much unexpected behavior.

Reply:
You think? I think people who were using it correctly (not giving the language code with the name) won't see a difference, right? You think there's people actually aiming for a name like "nl_nl_core_my_pipeline"? :|

Reply:
No, I think you might want `nl_nlcorpusname_something`.

Reply:
Yea, potentially. I actually had the check first like you suggested so I'll put it back like that.

Reply:
Do you still think it's potential breaking/unexpected then?

I actually found the "nl_nl_" bit unexpected myself :p

Reply:
I guess I'm not worried that it's unexpected, just that it's a change that could break things for existing workflows.

Reply:
There's always a chance, true :| 
We could put it behind an additional flag, a boolean whether to prepend lang, that is by default `True`? 
Or whatever, I'm not invested in this feature either 😁

Reply:
That seems like overkill, and I think the number of potentially affected users is really really small.